src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Security/customers.html.twig

```diff
@@ -0,0 +1,47 @@
+{#**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ *#}
+
+{% extends '@PrestaShop/Admin/layout.html.twig' %}
+
+{% block content %}
+  {% block security_list_panel %}
+    <div class="row">
+      <div class="col">
+        {% include '@PrestaShop/Admin/Common/Grid/grid_panel.html.twig' with {'grid': grid} %}
+      </div>
+    </div>
```

Comment:
```suggestion
        {% include '@PrestaShop/Admin/Common/Grid/grid_panel.html.twig' with {'grid': grid} %}
```

Reply:
To match with the huge work from @Hlavtox (example of one of the many PRs https://github.com/PrestaShop/PrestaShop/pull/26064/files) that aimed to clean useless bootstrap rows everywhere

Reply:
@jolelievre 

![FAEF1CD9-B541-4DC7-B4C6-4EAF9CCBFB64](https://user-images.githubusercontent.com/6097524/151437150-b5d3e9eb-bfd9-452a-8602-9e148474d9a2.jpeg)

Reply:
That's because my PR is one year old :trollface: